### PR TITLE
CRUD now working for companies

### PR DESCRIPTION
**Pull request by mikewesthub**
Set up controller for companies and refactored.

All tests are passing.

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  def show
+  end
+
+  def edit
+  end
+
+  def create
+    @company = Company.create(company_params)
```
Comment (rposborne) [inline app/controllers/companies_controller.rb]: Does this handle an invalid Company?

File: app/views/companies/create.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Companies#create</h1>
+<p>Find me in app/views/companies/create.html.erb</p>
```
Comment (rposborne) [inline app/views/companies/create.html.erb:2]: Should this be here?

File: app/views/companies/destroy.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Companies#destroy</h1>
+<p>Find me in app/views/companies/destroy.html.erb</p>
```
Comment (rposborne) [inline app/views/companies/destroy.html.erb:2]: Remove this

File: app/views/companies/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Companies#edit</h1>
+<p>Find me in app/views/companies/edit.html.erb</p>
```
Comment (rposborne) [inline app/views/companies/edit.html.erb]: Remove this.

File: app/views/companies/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Companies#index</h1>
+<p>Find me in app/views/companies/index.html.erb</p>
```
Comment (rposborne) [inline app/views/companies/index.html.erb]: What is this doing? Template code!?!!?

File: config/routes.rb [hunk 6]
```diff
@@ -2,7 +2,11 @@
 
   resources :people
   resources :interactions
+<<<<<<< HEAD
+  resources :companies
+=======
   resources :users
+>>>>>>> upstream/master
```
Comment (rposborne) [inline config/routes.rb]: MERGE CONFLICTS 💕

**Comment by sam-kim** [comment]
What a merge conflict!

**Pull request merged**